### PR TITLE
added removal of secrets in clean up script

### PR DESCRIPTION
**Pull request by alejandroEsc**
Removal of secrets as part of clean up script for external cluster.

### CODE REVIEW

**Comment by davidewatson** [comment]
I am okay merging this, though honestly, I think we should remove this script altogether. It makes assumptions which are changing (e.g. key names, deployment names, etc.)

**Comment by alejandroEsc** [comment]
Are they changing? If you say so, its just a script to get devs moving fast, you can set it to deprecate in the near future.

File: hack/clean-external-cluster.sh [hunk 1]
```diff
@@ -13,6 +13,8 @@ cluster=$2
 echo "Deleting machine object"
 kubectl patch machine ${machine} -p '{"metadata":{"finalizers": [null]}}'
 kubectl delete machine ${machine}
+kubectl delete secret master-private-key
```
Comment (davidewatson) [inline hack/clean-external-cluster.sh]: These key names are not referenced anywhere in the codebase. The existing template only defines a single key: https://github.com/samsung-cnct/cluster-api-provider-ssh/blob/8f19fd25d02a6a6586bac7433b5099f67cdca268/clusterctl/examples/ssh/provider-components.yaml.template#L118

Comment (alejandroEsc) [reply hack/clean-external-cluster.sh]: I didnt realize we consolidated those keys into one? interesting. alright i can update that.

**Pull request merged**